crates/topos/src/main.rs

```diff
@@ -29,8 +29,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     LogTracer::init()?;
 
     let args = options::Opt::parse();
-
-    let mut layers = Vec::new();
+    let jaeger_agent = args.jaeger_agent;
```

Comment:
We can rename every `jaeger` by `otlp` or `opentelemetry` I guess. We will not use `jaeger` anymore